app/models/views/applikation/result.rb

```diff
@@ -0,0 +1,28 @@
+module Views
+  module Applikation
```

Comment:
It can't be, we're using `Application` model all over. 

I've just tried to create this locally and it works: 
```ruby
module Views
  module Application
    class Something
    end
  end
end
```